src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown

```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
+
+While this is happening the screen will be white to the user and will look like it has stalled.
+This becomes especially apparent if you are loading your web page from the network where
+frequently many pages take more than 1 or 2 seconds to get any content visible on the homepage.
+
+To provide a better user experience you can control the color of the screen by adding a
+`background_color` to your manifest and giving it an HTML color value. The color will be used
+by Chrome the instant the web app is launched and will remain on the screen until the web
+apps first render.
+
+Simply set the following in your manifest.
+
+{% highlight json %}
+"background_color": "#2196F3",
+{% endhighlight %}
+
+There will now be no white screen as your site is launched from the homescreen.
+
+A good suggested value for this property is the background color of the load page.  Using the 
+same colors as the background page will allow for a smooth looking transistion from this
+splashscreen to the homepage.
+
+It is strongly suggested that you add the `background_color` in to your manifest.
+
+<div class="clear g-wide--full">
+    <figure class="fluid">
+        <img src="images/background-color.png" alt="backgroud color">
+
+        <figcaption>Background color for launch screen</figcaption>
+    </figure>
+</div>
+
+<div class="clear"></div>
+
+## Define a site-wide theme color
+
+Chrome introduced the concept of a theme color for your site in 2014.  The theme color
+is a hint from your web page that tells the browser what color to tint
+ [UI elements such as the address bar](additional-customizations.html).  
+
+<div class="clear g-wide--full">
+    <figure class="fluid">
+        <img src="images/theme-color.png" alt="backgroud color">
+
+        <figcaption>Theme color</figcaption>
+    </figure>
+</div>
+
+The problem is, you have to define the theme color on every single page, and if 
+you have a large site or legacy site, making a lot of site wide changes is not feasible.
+
+Add in a `theme_color` attribute to your manifest, and when the site is launched
+from the homescreen every page in the domain will automatically have the theme color
+applied.
+
+{% highlight json %}
+"theme_color": "#2196F3"
+{% endhighlight %}
+
+It is suggested that you add the `theme_color` in.
+
+<div class="clear g-wide--full">
+    <figure class="fluid">
+        <img src="images/manifest-display-options.png" alt="backgroud color">
+
+        <figcaption>Sitewide theme color</figcaption>
+    </figure>
+</div>
+
 ## Define the initial orientation of the page
 
-You can enforce a specific orientation, which is really useful for some use cases like games, which may only work in landscape. However, this should be used with care. Users prefer being able to view apps in both orientations.
+You can enforce a specific orientation, which is really useful for some use cases like games,
+which may only work in landscape. However, this should be used with care. Users prefer being
```

Comment:
Suggest change to 'which may only work in one orientation. Use this with care. Users prefer selecting the orientation.'